### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by eydun**
Added explanation for ignored text-nodes in reponse.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -285,8 +285,8 @@ <h3 id="targeting-response">Targeting the response</h3>
 
       <p>
         By default <code>&lt;i-html></code> will parse the response as HTML,
-        and inject the <code>&lt;body></code> element's contents into itself.
-        This can be changed by altering the <code>target=</code> attribute on
+        and inject the <code>&lt;body></code> element's contents into itself, ignoring any unwrapped test nodes.
+	      The selection of content to be injected can be customized by setting the <code>target=</code> attribute on
```
Comment (keithamus) [inline index.html]: Could we wrap these lines please?

File: index.html [hunk 2]
```diff
@@ -681,8 +681,8 @@ <h4 id="sanitization">Sanitization</h4>
         <dt><code>allow="media"</code></dt>
         <dd>
           This allows the rendering of <code>&lt;img></code>,
-          <code>&lt;picture></code>, <code>&lt;video></code> 
-          <code>&lt;audio></code>, and <code>&lt;object></code> elements in the
+          <code>&lt;picture></code>, <code>&lt;video></code>,
+          <code>&lt;audio></code> and <code>&lt;object></code> elements in the
```
Comment (keithamus) [inline index.html]: I prefer oxford commas if we could please.
```suggestion
          <code>&lt;audio></code>, and <code>&lt;object></code> elements in the
```
But this line will also need wrapping I think?

**Review by keithamus — APPROVED** [review]
Thanks for this and the various fixups. A couple of tiny nits

File: index.html (hunk 2 above)
Comment (eydun) [reply index.html]: > I prefer oxford commas if we could please.

Sure, I was not actually aware of the Oxford comma until now 🙂

File: index.html (hunk 1 above)
Comment (eydun) [reply index.html]: I am not sure, what is causing this... I am using PHPStorm on Windows.

**Review by keithamus — APPROVED** [review]

**Pull request merged**